index.html

```diff
@@ -12,6 +12,7 @@
     />
     <link rel="stylesheet" href="style.css"/>
     <title>課題</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
```

Comment:
こういう風にstylesheetはまとめておいてください。
```suggestion
    <link rel="stylesheet" href="style.css"/>
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
    <title>課題</title>
```

Reply:
わかりました！

Reply:
すみません、書き方間違ってました...
自分が作ったcss一番下にしましょう。（優先度的に）

Reply:
わかりました！